kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/flight/FlightFinder.java

```diff
@@ -1,13 +1,28 @@
 package com.kodilla.good.patterns.flight;
 
-import java.util.ArrayList;
-import java.util.List;
+import java.util.Map;
 
-public class FlightResult {
+public class FlightFinder {
 
-    private final List<Flight> flights = new ArrayList<>();
+    public void findFlightFromAirport(String departureAirport, Map<Integer, Flight> availableFlightMap) {
```

Comment:
`availableFlightMap` nie powinno być przekazywane do tej metody jako argument. To jest "baza danych lotów", więc powinna być wstrzyknięta do klasy wyszukiwarki jako zależność i wywoływana wewnątrz metody `flindFlightFromAirport`. Ta sama uwaga dla 2 kolejnych metod.

Reply:
2. Metoda typu `find` powinna zwracać znaleziony lot (jeszcze lepiej: listę znalezionych lotów). Nie powinna być typu `void` - jeśli w tej konkretnej aplikacji chcesz wypisać wynik na konsolę, to zrób to w metodzie `main`, a nie tutaj. Ta sama uwaga do pozostałych metod tej klasy.